deployment/template-cronjob.yaml

```diff
@@ -129,13 +129,13 @@ parameters:
   displayName: Days for which report needs to be run
   required: true
   name: DAYS
-  value: "7"
+  value: "1"
 
 - displayName: "Model bucket Name"
   description: Name of the bucket which contains the model
   required: true
   name: AWS_BUCKET_NAME_MODEL
-  value: "stage-openshift-probable-vulnerability-models"
+  value: "openshift-probable-vulnerability-models"
```

Comment:
We don't require bucket parameter now, as we are reading value from secret. I deleted the same from my PR.

Reply:
Sure, I can fix it.

Reply:
you can simply undo this changes...it will be taken care by my PR.

Reply:
I totally forgot that it is been changed to use secrets :P lol.